app/models/katello/host/content_facet.rb

```diff
@@ -18,6 +18,8 @@ class ContentFacet < Katello::Model
       validates :host, :presence => true, :allow_blank => false
       validates_with Validators::ContentViewEnvironmentValidator
 
+      attr_accessible :content_view, :content_view_id, :lifecycle_environment, :lifecycle_environment_id, :host
```

Comment:
@daviddavis While foreman is using protected attributes, what do you think the best way to avoid mass assignment errors for these attributes would be?